src/host/srvinit.cpp

```diff
@@ -428,11 +428,6 @@ HRESULT ConsoleCreateIoThread(_In_ HANDLE Server,
                                               [[maybe_unused]] PCONSOLE_API_MSG connectMessage)
 try
 {
-    // Create a telemetry instance here - this singleton is responsible for
-    // setting up the g_hConhostV2EventTraceProvider, which is otherwise not

```

Comment:
who sets this up now?

Reply:
`src/host/exe/exemain.cpp` here: https://github.com/microsoft/terminal/pull/16253/files#diff-47e1cca9863ea63bf26633f56e642bf72560cec16f786631cec4f3d9d885a073R225

Reply:
So... about that `TraceLoggingRegister` call...
mhm

Reply:
_yup!_

How did you notice?